tests/data/otel-collector-config.yaml

```diff
@@ -5,6 +5,10 @@ receivers:
   otlp:
     protocols:
       grpc:
+        tls:
+          ca_file: "certs/ca.pem"
+          cert_file: "certs/server-cert.pem"
+          key_file: "certs/server-key.pem"
```

Comment:
I'm not sure if this is testing mTLS:

```
Receivers leverage server configuration.

Beyond TLS configuration, the following setting can optionally be configured (required for mTLS):

    - client_ca_file: Path to the TLS cert to use by the server to verify a client certificate. (optional) This sets the ClientCAs and ClientAuth to RequireAndVerifyClientCert in the TLSConfig. Please refer to https://godoc.org/crypto/tls#Config for more information.
    - client_ca_file_reload (default = false): Reload the ClientCAs file when it is modified.
```

https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md#server-configuration

Is this in purpose?

Reply:
you are right, it's missing the client ca file